modifytags/modifytags_test.go

```diff
@@ -0,0 +1,108 @@
+package modifytags
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"go/format"
+	"go/parser"
+	"go/token"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+var update = flag.Bool("update", false, "update golden (.out) files")
+
+// This is the directory where our test fixtures are.
+const fixtureDir = "./test-fixtures"
```

Comment:
(no action required)

FWIW, this should probably be /testdata/. But let's leave it as-is.